src/static/about.html

```diff
@@ -137,15 +137,22 @@ <h2>Open-Source Licenses</h2>
   </body>
   <script>
     const { ipcRenderer, shell } = require('electron');
+    const { getLogger } = require('../logger');
+    const logger = getLogger(__filename.split('/').slice(-1)[0]);
+
     const node1 = document.getElementById('invest-version');
     const node2 = document.getElementById('workbench-version');
-    ipcRenderer.on('variable-reply', (event, arg) => {
-      const text1 = document.createTextNode(arg.investVersion)
+
+    ipcRenderer.invoke('variable-request')
+    .then(response => {
+      const text1 = document.createTextNode(response.investVersion)
       node1.appendChild(text1)
-      const text2 = document.createTextNode(arg.workbenchVersion)
+      const text2 = document.createTextNode(response.workbenchVersion)
       node2.appendChild(text2)
     })
-    ipcRenderer.send('variable-request', 'ping');
+    .catch(e => {
+        logger.error("Error processing versions for About page");
+    });
```

Comment:
Here's one I probably don't mind catching, because if the `About` page is missing some of it's data, it's no big deal. But I'd prefer logging the actual error message. I think it's in `e.message`.

Reply:
Good point, `e.message` it is: https://github.com/natcap/invest-workbench/pull/135/commits/6be3fcc4ec592f9cc8366d654a464439f8b1e245